proxy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/python-311
+FROM registry.access.redhat.com/ubi9/python-311@sha256:fae422b4e7dbc2e551919fb87b6004c06b731179be2ed2904427531a2d1ef3a2
```

Comment:
do we want to start using `sha` from now on?

Reply:
I got docker file lint error llike this: `-:1 DL3006 warning: Always tag the version of an image explicitly`.  So I put the `sha`.

Reply:
Do they also tag images or do we have to use the sha here? 
(I don't have a strong preference, just haven't worked with the UBI registry much before)

Reply:
yeah, me neither, that's why I'm asking, I'm totally fine with this approach if it has sense with what we are using

Reply:
@psschwei They have tag like this `1-62.1717085982`.  It doesn't look user friendly, either.